src/bin/add/args.rs

```diff
@@ -4,42 +4,103 @@ use cargo_edit::Dependency;
 use cargo_edit::{get_latest_dependency, CrateName};
 use semver;
 use std::path::PathBuf;
+use structopt::StructOpt;
 
 use crate::errors::*;
 
-#[derive(Debug, Deserialize)]
-/// Docopts input args.
+#[derive(Debug, StructOpt)]
+#[structopt(bin_name = "cargo")]
+pub enum ArgsWrap {
+    #[structopt(name = "add", author = "")]
+    #[structopt(
+        after_help = "This command allows you to add a dependency to a Cargo.toml manifest file. If <crate> is a github
+or gitlab repository URL, or a local path, `cargo add` will try to automatically get the crate name
+and set the appropriate `--git` or `--path` value.
+
+Please note that Cargo treats versions like '1.2.3' as '^1.2.3' (and that '^1.2.3' is specified
+as '>=1.2.3 and <2.0.0'). By default, `cargo add` will use this format, as it is the one that the
+crates.io registry suggests. One goal of `cargo add` is to prevent you from using wildcard
+dependencies (version set to '*')."
+    )]
+    /// Add dependency to a Cargo.toml manifest file.
```

Comment:
I think the recommended way is to put docs above attributes. Could you change that here and below?